tests/data_context/test_data_context_variables.py

```diff
@@ -498,18 +523,27 @@ def test_file_data_context_variables_e2e(
     It is also important to note that in the case of $VARS syntax, we NEVER want to persist the underlying
     value in order to preserve sensitive information.
     """
-    # Prepare updated progress bars to set and serialize to disk
+    # Prepare updated progress_bars to set and serialize to disk
     updated_progress_bars: ProgressBarsConfig = copy.deepcopy(progress_bars)
     updated_progress_bars.globally = False
     updated_progress_bars.profilers = True
 
+    # Prepare updated include_rendered_content to set and serialize to disk
+    updated_include_rendered_content: IncludeRenderedContentConfig = copy.deepcopy(
+        include_rendered_content
+    )
+    updated_include_rendered_content.expectation_validation_result = True
```

Comment:
This is an E2E test so once we set this once, all subsequent changes are just going to change `True` to `True`. Is there value in that?

Reply:
Per our offline conversation this is not persistent.